### PR TITLE
MPA-10: Support Multilanguage

### PR DESCRIPTION
**Pull request by DaniilGirskiy**

### CODE REVIEW

**Comment by pedroseruca** [comment]
Hey! can you change the commit name to `MPA-10: Added support for localizable`? I suggest to have the ticket we work on as a prefix to help on history when PR gets merged

File: MinderaPeople/Screens/LoginView.swift [hunk 1]
```diff
@@ -96,7 +96,8 @@ struct LoginView: View {
                         .foregroundColor(.accentColor)
                     
                     Spacer()
-                    MinderaButton(.title("Login with Google Account")) {
+                    MinderaButton(.title(String(localized: "login",
+                                                table: "LoginLocalizable"))) {
```
Comment (pedroseruca) [inline MinderaPeople/Screens/LoginView.swift:100]: not a particular fan of having this on the view... maybe we should have a localisation service... not sure the best approach to fit this although... maybe @p4checo can give us a hint 🙏🏻

Comment (p4checo) [reply MinderaPeople/Screens/LoginView.swift:100]: hi both 👋🏼 

I have never done localization in SwiftUI, but I don't think we need to create a strings file for each feature... Normally you have one `Localizable.strings` file _per module_.

There are plenty of examples [[1](https://stackoverflow.com/questions/58578341/how-to-implement-localization-in-swift-ui)], tutorials [[1](https://www.kodeco.com/27469286-swiftui-localization-tutorial-for-ios-getting-started)][[2](https://www.ibabbleon.com/swiftui_localization_tutorial.html)] and documentation [[1](https://developer.apple.com/documentation/swiftui/preparing-views-for-localization)][[2](https://developer.apple.com/documentation/swiftui/text/init(_:tablename:bundle:comment:))] about adding localization in SwiftUI... Perhaps you could take a look? 😄

I think that by default the system looks for strings in a `Localizable.strings` file, not some "random" file - that's probably why you had to resort to using the `table` parameter in the first place. 

I agree that specifying the strings dictionary on every call is not ideal, and ideally we should either use native `Text("Do something")` where "Do Something" will be the localization key, or we could use identifiers like `"my_feature.do_something_button.cta"` which would then be used as the key. 

If **needed** we can then create helpers like a computed property `var localized: LocalizedStringKey` that automatically converts a string to its localized counterpart (e.g. if we need to pass a localized string to an "external" module, like MinderaDesignSystem`).

I don't think we need anything like a service, as the system should handle everything for us if we have thing properly set up. 

Hope this helps 🙏🏼

**Comment by beloso** [comment]
Do you guys feel we could have some helpers for Views/ViewBuilds that would receive as Input a `LocalizableString`(Protocol), that would handle translating the value into the I18N key. Then our localization enums would to comply with `LocalizableString`, and this view would be the exact same as the native component, but it would do the String mapping automatically. 

Otherwise we will always have to add the `.rawValue` when passing on localizable strings. But this is mostly a nice to have and can be a future enhancement.

**Pull request closed without merging**